Automattic/wp-calypso
# Jetpack Cloud: vertically align header icons, make Jetpack logo size consistent

#### Changes proposed in this Pull Request

* For sites that do not have Scan, add vertical padding to the Scan "security" icon so that it lines up with the layout of the Backup page for sites that don't have Backup.
* Adjust route testing in the master bar, so that we recognize `/backup` as a non-exterior page and show the correct Jetpack logo size for mobile layouts.

Fixes `p1590064350316900-slack-jetpack-voyager`

#### Implementation notes

* I'm still not sure the master bar route test is entirely correct; see my inline comment below.

#### Testing instructions

* Open Jetpack Cloud and select a site without Scan or Backup.
* Navigate back and forth between the "Latest backups" and "Scanner" pages, ensuring that the header icons and copy don't shift around.
* Observe that on mobile layouts, the Jetpack icon in the upper left is the same on the "Latest backups" page as it is on the "Scanner" page.

#### Screenshots

![Kapture 2020-05-21 at 10 12 27](https://user-images.githubusercontent.com/670067/82574385-a1a53500-9b4c-11ea-9272-fa230575d8cd.gif)
